app/views/assignments/rotation_generators/prompt.html.haml

```diff
@@ -0,0 +1,27 @@
+%h1 Generate rotation
+
+.container-fluid
+  = form_with model: @generator,
+              url: roster_assignments_generate_rotation_path(@roster),
+              local: true do |f|
+    .row
+      .col-lg
+        .mb-3
+          = f.label :start_date, nil, class: 'form-label'
+          = f.date_field :start_date, class: 'form-control'
+        .mb-3
+          = f.label :end_date, nil, class: 'form-label'
+          = f.date_field :end_date, class: 'form-control'
+        .mb-3
+          = f.label :starting_user_id, 'Starting with', class: 'form-label'
+          = f.collection_select :starting_user_id, @roster.users.active.order(:last_name),
+                                :id, :last_name, {}, class: 'form-select'
+      .col-lg
+        .mb-3
+          = f.label :user_ids, 'Users', class: 'form-label'
+          = f.select :user_ids,
+                     options_from_collection_for_select(@roster.users.active.order(:last_name),
+                                                        :id, :last_name, @roster.users.pluck(:id)),
+                     { include_hidden: false },
+                     multiple: true, size: 10, class: 'form-select', selected: @roster.users.active
```

Comment:
```suggestion
                     multiple: true, size: 10, class: 'form-select'
```
This was previously an ok way to set the default as the previous form didn't exactly handle errors very well, however...

Now this means that upon validation error, any changes made to this field will be wiped out. A better way to set this default would be in the controller with the others.